src/base/static/components/atoms/typography.js

```diff
@@ -211,6 +215,12 @@ Link.propTypes = {
   href: PropTypes.string.isRequired,
 };
 
+const Time = props => <time>{moment(props.time).fromNow()}</time>;
```

Comment:
We may want to pass in a datetime attribute here, perhaps something like `<time datetime={moment(props.time).format('YYYY-MM-DD')}>`

 https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time
http://momentjs.com/docs/#/displaying/format/

Reply:
I don't think we are misusing `time` - we just aren't using it to it's full abilities. Whatever we pass in to the element's child is purely presentational, so `fromNow()` is fine, but we should also be passing in the `datetime` attribute as well.

I wasn't being thorough in my example above, but it would look like this:

`<time datetime={moment(props.time).format('YYYY-MM-DD')}>{moment(props.time).fromNow()}</time>`